rules/no-skip-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+module.exports = function (context) {
+	var ava = createAvaRule();
+	return ava.merge({
+		MemberExpression: function (node) {
+			if (ava.isTestFile && node.property.name === 'skip' && util.isCallFromTestObject(node)) {
```

Comment:
@jfmengels See @twada's example. You need to use a `CallExpression` hook instead.